aws/resource_aws_opsworks_stack_test.go

```diff
@@ -505,15 +522,28 @@ func testAccCheckAwsOpsworksStackDestroy(s *terraform.State) error {
 func testAccAwsOpsworksStackConfigNoVpcCreate(name string) string {
 	return fmt.Sprintf(`
 provider "aws" {
-  region = "us-east-1"
```

Comment:
I think the rest of these `us-east-1` removals (e.g. all the "no VPC" tests) is fine and we should just use the normal region with no provider configuration at all.

Reply:
Removed these all

Reply:
😍  thank you